Bottom: 9d5e87ec3982847e89b8102eccac7589d928ca56
Top:    36c2f569c4b8537fb0c0830bef0761725bb3197f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 14:48:42 +0000

Refresh of w32-update-pango-recipe-to

---

diff --git a/w32/recipes/pango.recipe b/w32/recipes/pango.recipe
index 5cd9212..efa36a6 100644
--- a/w32/recipes/pango.recipe
+++ b/w32/recipes/pango.recipe
@@ -1,9 +1,11 @@
 name=pango
-version=1.26.0
+major=1.28
+minor=3
 release=1
-download=http://ftp.gnome.org/pub/gnome/sources/pango/1.26
+version=$major.$minor
+download=http://ftp.gnome.org/pub/gnome/sources/$name/$major
 
 configure_opt="
   --with-included-modules
-  CXX=${host:+$host-g++}  
+  CXX=${host:+$host-g++}
 "
